### PR TITLE
Update GOV.UK Verify contact details

### LINKED ISSUE #373: Remove personal email addresses
The manual shouldn't contain email addresses for individuals. They're too fragile and easy to break, and we probably wouldn't notice for a long time.

| Email | Mailing list |
| ------ | ------------ |
| Nayeema.Chowdhury | performance |
| janet.hughes | ? |

It would be nice if we had some way to periodically test that the email addresses we include are valid, but I can't think of a nice automated way to do that.

### PR DESCRIPTION
**Pull request by daibach**
Resolves #373

### CODE REVIEW

File: service-manual/identity-assurance/index.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Each relying party will need to go through the programme’s induction process.
 3. Planning -- complete a readiness assessment. Agree a detailed plan setting out what the service and GDS will each do and when. Get internal approvals within GDS and the relying party's organisation.
 4. Connect to hub -- agree a memorandum of understanding between the GOV.UK Verify Programme and the department responsible for the service.
 
-Government providers of digital services that want to find out more about GOV.UK Verify, with a view to becoming relying parties, should contact [Janet Hughes](mailto:janet.hughes@digital.cabinet-office.gov.uk).
+Government providers of digital services that want to find out more about GOV.UK Verify, with a view to becoming relying parties, should contact [Jess McEvoy](mailto:govukverify_engagement@digital.cabinet-office.gov.uk).
```
Comment (bradwright) [inline service-manual/identity-assurance/index.md]: Would it be better to also remove the name from this? The email address is role-based.

Comment (daibach) [inline service-manual/identity-assurance/index.md]: I was considering that, but for now they'd like a named contact in there.

Comment (bradwright) [inline service-manual/identity-assurance/index.md]: :+1:

**Comment by bradwright** [comment]
Merge when green.

**Pull request merged**